data/crac-io/crac-io-cne/src/main/java/com/farao_community/farao/data/crac_io_cne/CneClassCreator.java

```diff
@@ -58,4 +66,119 @@ public static TimeSeries newTimeSeries(String businessType, String curveType, Se
+    }
+
+
+    /*****************
+     MONITORED SERIES
+     *****************/
+    public static MonitoredSeries newMonitoredSeries(String id, String name) {
+        MonitoredSeries monitoredSeries = new MonitoredSeries();
+        monitoredSeries.setMRID(cutString(id, 60));
+        monitoredSeries.setName(name);
+        monitoredSeries.registeredResource = new ArrayList<>();
+
+        return monitoredSeries;
+    }
+
+    public static MonitoredSeries newMonitoredSeries(String id, String name, MonitoredRegisteredResource monitoredRegisteredResource) {
+        MonitoredSeries monitoredSeries = newMonitoredSeries(id, name);
+        monitoredSeries.registeredResource = Collections.singletonList(monitoredRegisteredResource);
+
+        return monitoredSeries;
+    }
+
+    /*****************
+     MONITORED REGISTERED RESOURCE
+     *****************/
+    public static MonitoredRegisteredResource newMonitoredRegisteredResource(String id, String name, String inAggregateNodeMRID, String outAggregateNodeMRID) {
+        MonitoredRegisteredResource monitoredRegisteredResource = new MonitoredRegisteredResource();
+        monitoredRegisteredResource.setMRID(createResourceIDString(A02_CODING_SCHEME, id));
+        monitoredRegisteredResource.setName(name);
+        monitoredRegisteredResource.setInAggregateNodeMRID(createResourceIDString(A02_CODING_SCHEME, inAggregateNodeMRID));
+        monitoredRegisteredResource.setOutAggregateNodeMRID(createResourceIDString(A02_CODING_SCHEME, outAggregateNodeMRID));
+        monitoredRegisteredResource.measurements = new ArrayList<>();
+
+        return monitoredRegisteredResource;
+    }
+
+    public static MonitoredRegisteredResource newMonitoredRegisteredResource(String id, String name, String inAggregateNodeMRID, String outAggregateNodeMRID, List<Analog> measurementsList) {
+        MonitoredRegisteredResource monitoredRegisteredResource = newMonitoredRegisteredResource(id, name, inAggregateNodeMRID, outAggregateNodeMRID);
+        monitoredRegisteredResource.measurements = measurementsList;
+
+        return monitoredRegisteredResource;
+    }
+
+    /*****************
+     MEASUREMENT
+     *****************/
+    public static Analog newMeasurement(String measurementType, Unit unit, double flow) {
+        Analog measurement = new Analog();
+        measurement.setMeasurementType(measurementType);
+
+        if (unit.equals(Unit.MEGAWATT)) {
+            measurement.setUnitSymbol(MAW_UNIT_SYMBOL);
+        } else if (unit.equals(Unit.AMPERE)) {
+            measurement.setUnitSymbol(AMP_UNIT_SYMBOL);
+        } else {
+            throw new FaraoException(String.format("Unhandled unit %s", unit.toString()));
+        }
+
+        if (flow < 0) {
+            measurement.setPositiveFlowIn(OPPOSITE_POSITIVE_FLOW_IN);
+        } else {
+            measurement.setPositiveFlowIn(DIRECT_POSITIVE_FLOW_IN);
+        }
+        measurement.setAnalogValuesValue(limitFloatInterval(flow));
```

Comment:
why do you need the function limitFloatInterval here?

Reply:
To avoid very big values (infinite), that are irrelevant, and non-xsd compliant because they are written in scientific notation.

Reply:
ok